Learn-JSON-Fetch-async-await-building-fcc-author-page/index.js

```diff
@@ -0,0 +1,45 @@
+const authorContainer = document.getElementById("author-container");
+const loadMoreBtn = document.getElementById("load-more-btn");
+let startingIndex = 0;
+let endingIndex = 8;
+let authorDataArr;
+
+// replace this API with one hosted by FCC
+fetch("https://fcc-author-api.herokuapp.com/authors")
+  .then((res) => res.json())
+  .then((data) => {
+    authorDataArr = data;
+    displayUsers(authorDataArr.slice(startingIndex, endingIndex));
+  })
+  .catch((err) => {
+    authorContainer.innerHTML = `<p class="error-msg">There was an error loading the data</p>`;
+    console.error(`There was an error: ${err}`);
+  });
+
+const fetchMoreAuthors = () => {
+  startingIndex += 8;
+  endingIndex = endingIndex + 8;
```

Comment:
Is there a specific reason not to use addition assignment like we did above?

Reply:
Yeah, that is a good point. 
I can change that real quick 👍